src/server/recsys/test/recommender-tests.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+/**************************************************************
+ * 
+ * UNIT TESTS FOR RECOMMENDER ENGINE 
+ *
+ */
+
+
+let assert = require('assert');
+let x5recommend = require('../engine/x5recommend.js');
+let path = require('path');
+
+const qm = require('qminer');
+
+// internal modules
+const Logger = require('../../../lib/logging-handler')();
+
+// create a logger instance for logging recommendation requests
+const logger = Logger.createGroupInstance('recommendation-requests', 'x5recommend');
```

Comment:
Would be better to name 'recommendation-requests-tests'?

Reply:
I see. I checked and the log file name is already used [here](https://github.com/JozefStefanInstitute/x5gon/blob/master/src/server/recsys/engine/x5recommend.js#L15). Using the same file name would then (probably) merge both logs from unit tests and from methods within the `x5recommend` class. 

I suggest to rename `recommendation-requests` to `recommendation-tests`, if you agree.